rules-reviewed/jboss-eap/websphere/WebSphere-other.windup.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ruleset id="websphere-other"
+    xmlns="http://windup.jboss.org/schema/jboss-ruleset"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+
+    <metadata>
+        <description>
+            Provides analysis of WebSphere proprietary classes and constructs that may require individual attention when migrating to JBoss EAP 6+.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.2.0.Final"/>
+            <addon id="org.jboss.windup.rules,windup-rules-java,2.2.0.Final"/>
+        </dependencies>
+        <sourceTechnology id="websphere" versionRange="(10,12]"/>
```

Comment:
Failed to parse XML configuration at: file:/home/ondra/work/Migration/windup-rulesets/rules-reviewed/jboss-eap/websphere/WebSphereMQ.windup.xml due to: The 'sourceTechnology' element must have a non-empty 'versionRange' attribute